.prettierignore

```diff
@@ -0,0 +1,25 @@
+frontend/
```

Comment:
I guess these are here so that if we run prettier from the root, the submodules will not be affected?

Reply:
Yes. [Combining the  config files](https://github.com/eslint/eslint/discussions/16960) is also an option, but I don't think I have time for that any longer.

Reply:
Yes. The config files could probably be combined if we wanted to and then there would be no need to specify this, but then some of the svelte dependencies would have to be downloaded to the root, and that is probably not something we want.